README.md

```diff
@@ -166,543 +119,10 @@ There are two parameters:
-
-	if err != nil {
-		log.Fatalf("connection: %s", err)
-		return
-	}
-
-	cfg := queue.Cfg{
-		Temporary:  true,
-		IfNotExists: true,
-		Kind:       queue.FIFO,
-		Opts: queue.Opts{
-			Ttl:   10 * time.Second,
-			Ttr:   5 * time.Second,
-			Delay: 3 * time.Second,
-			Pri:   1,
-		},
-	}
-
-	que := queue.New(conn, "test_queue")
-	if err = que.Create(cfg); err != nil {
-		log.Fatalf("queue create: %s", err)
-		return
-	}
-
-	// put data
-	task, err := que.Put("test_data")
-	if err != nil {
-		log.Fatalf("put task: %s", err)
-	}
-	fmt.Println("Task id is", task.Id())
-
-	// take data
-	task, err = que.Take() //blocking operation
-	if err != nil {
-		log.Fatalf("take task: %s", err)
-	}
-	fmt.Println("Data is", task.Data())
-	task.Ack()
-
-	// take typed example
-	putData := customData{}
-	// put data
-	task, err = que.Put(&putData)
-	if err != nil {
-		log.Fatalf("put typed task: %s", err)
-	}
-	fmt.Println("Task id is ", task.Id())
-
-	takeData := customData{}
-	//take data
-	task, err = que.TakeTyped(&takeData) //blocking operation
-	if err != nil {
-		log.Fatalf("take take typed: %s", err)
-	}
-	fmt.Println("Data is ", takeData)
-	// same data
-	fmt.Println("Data is ", task.Data())
-
-	task, err = que.Put([]int{1, 2, 3})
-	task.Bury()
-
-	task, err = que.TakeTimeout(2 * time.Second)
-	if task == nil {
-		fmt.Println("Task is nil")
-	}
-
-	que.Drop()
-}
-```
-Features of the implementation:
-
-- If you use connection timeout and call `TakeWithTimeout` with parameter greater than the connection timeout then parameter reduced to it
-- If you use connection timeout and call `Take` then we return a error if we can not take task from queue in a time equal to the connection timeout
-
-## Multi connections
-
-You can use multiple connections config with tarantool/multi. 
-
-Main features:
-
-- Check active connection with configurable time interval and on connection fail switch to next in pool.
-- Get addresses list from server and reconfigure to use in MultiConnection.
+## Contributing
 
-Additional options (configurable via `ConnectWithOpts`):
-
-* `CheckTimeout` - time interval to check for connection timeout and try to switch connection
-* `ClusterDiscoveryTime` - time interval to ask server for updated address list (works on with `NodesGetFunctionName` set)
-* `NodesGetFunctionName` - server lua function name to call for getting address list
-
-## Tests
-
-You need to [install Tarantool](https://www.tarantool.io/en/download/) to run tests.
-See [Installation](#installation) section for requirements.
-
-To install test dependencies (like [tarantool/queue](https://github.com/tarantool/queue) module), run
-```bash
-make deps
-```
-
-To run tests for the main package and each subpackage, call
-```bash
-make test
-```
-Tests set up all required `tarantool` processes before run and clean up after.
-
-If you want to run a specific package tests, go to a package folder
-```bash
-cd multi
-```
-and call
-```bash
-go clean -testcache && go test -v
-```
```

Comment:
Use the same for the main package `tarantool` and the subpackages `queue` and `uuid`.

Reply:
Looks not relevant for proposed changes.